src/Microsoft.Sbom.Api/SBOMGenerator.cs

```diff
@@ -51,26 +54,25 @@ public SBOMGenerator(StandardKernel kernel, IFileSystemUtils fileSystemUtils)
             string componentPath,
             SBOMMetadata metadata,
             IList<SBOMSpecification> specifications = null,
-            RuntimeConfiguration configuration = null,
+            RuntimeConfiguration runtimeConfiguration = null,
             string manifestDirPath = null,
             string externalDocumentReferenceListFile = null)
         {
             // Get scan configuration
-            var config = configurationBuilder.GetConfiguration(
+            var inputConfiguration = ApiConfigurationBuilder.GetConfiguration(
                 rootPath,
                 manifestDirPath, null, null, metadata, specifications,
-                configuration, externalDocumentReferenceListFile, componentPath);
+                runtimeConfiguration, externalDocumentReferenceListFile, componentPath);
 
             // Initialize the IOC container. This varies depending on the configuration.
```

Comment:
Is this comment still relevant? It seems like it's configuration initialization, not IoC container initialization